pymongo/mongo_client.py

```diff
@@ -730,6 +740,7 @@ def __init__(
             server_selector=options.server_selector,
             heartbeat_frequency=options.heartbeat_frequency,
             fqdn=fqdn,
+            srv_service_name=srv_service_name,
```

Comment:
Please add a test to make sure that srvServiceName is supported via kwargs as well as in the URI. For example:
```python
client = MongoClient('mongodb+srv://...', srvServiceName='namefromkwarg')
self.assertEqual(client's srvServiceName, 'namefromkwarg')
```

Reply:
Done.